File: ABC/062/A.py
import sys
x,y = map(int,input().split())

if x <= 0 or x >= 13 or y <= 0 or y >= 13:
    sys.exit()
if x >= y:
    sys.exit()

A = (1, 3, 5, 7, 8, 10, 12)
B = (4, 6, 9, 11)

if x == 2 or y == 2:
    print("No")
    sys.exit()
if x in A and y in A:
    print("Yes")
elif x in B and y in B:
    print("Yes")
else:
    print("No")

File: ABC/ruby/062/A.rb
x = []
x = gets().chomp.split(' ').map { |n| n.to_i }

exit if !( 1 <= x[0] && x[0] <= 12 && 1 <= x[1] && x[1] <= 12 && x[0] < x[1] )

group_1 = [1,3,5,7,8,10,12]
group_2 = [4,6,9,11]

# puts (group_1 & x).length
puts (group_1 & x).length == 2 || (group_2 & x).length == 2  ? 'Yes' : 'No' 
# print if group_1.include?(%w[x y]) || group_2.include?(%w[x y]) ? 'Yes' : 'No'